### PR TITLE
update to linter v2

### PR DESCRIPTION
**Pull request by dirk-thomas**
Upgrading to Standard Linter v2.

### CODE REVIEW

File: lib/init.coffee [hunk 1]
```diff
@@ -252,14 +252,17 @@ module.exports =
           messages = @parseSchemaMessages(textEditor, output)
         if messages.length
           for message in messages
-            message.type = 'Error'
-            message.text = message.text + ' (' + schemaUrl + ')'
-            message.filePath = textEditor.getPath()
+            message.severity = 'error'
+            message.excerpt = message.excerpt + ' (' + schemaUrl + ')'
```
Comment (dirk-thomas) [inline lib/init.coffee]: `schemaUrl` might contain a relative path and not necessarily an absolute URL. I am not sure if `message.url` works with that? If not I would rather keep it as is for now and keep this improvement for another PR.

**Comment by dirk-thomas** [comment]
I bumped the versions in 478b9678c404bd0572fb7dc411708450c928914f.

File: lib/init.coffee (hunk 1 above)
Comment (Arcanemagus) [inline lib/init.coffee]: Ah, I just assumed it was a URL from the name. Note that in v2 messages `message.excerpt` is only rendered as text, links are not clickable.

Comment (dirk-thomas) [inline lib/init.coffee]: Maybe some heuristic: if `schemaUrl` contains `://` then put it into `message.url`, otherwise keep it where it is?

Comment (Arcanemagus) [inline lib/init.coffee]: That makes sense to me if it can contain both possibilities.

Comment (dirk-thomas) [inline lib/init.coffee]: Here we go: 22ba410e17589004189cc4af4a8567ca34dc72a7

Comment (dirk-thomas) [inline lib/init.coffee]: ~~Here we go: 22ba410e17589004189cc4af4a8567ca34dc72a7~~

And hopefully making the linter happy: 0951606aae680872e27d1279e8b35c6d88d482c6

File: lib/init.coffee [hunk 2]
```diff
@@ -252,14 +252,20 @@ module.exports =
           messages = @parseSchemaMessages(textEditor, output)
         if messages.length
           for message in messages
-            message.type = 'Error'
-            message.text = message.text + ' (' + schemaUrl + ')'
-            message.filePath = textEditor.getPath()
+            message.severity = 'error'
+            if schemaUrl.indexOf('://') isnt -1
```
Comment (Arcanemagus) [inline lib/init.coffee]: `schemaUrl.includes('://')` is probably clearer here.

Comment (dirk-thomas) [inline lib/init.coffee]: Fancy coffeescript stuff :wink: 9c64325c7b911cdb250670ddb6bd8409f0bca84d

Comment (Arcanemagus) [inline lib/init.coffee]: *Javascript: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

Comment (dirk-thomas) [inline lib/init.coffee]: Doh, obviously not my daily language :smile:

Thanks for the feedback!
